content/tracing/advanced_usage/priority_sampling.md

```diff
@@ -40,9 +40,31 @@ public class MyClass {
 ```
 {{% /tab %}}
 {{% tab "Python" %}}
+To enable sampling priority in the tracer:
 ```python
-This is some python code
+tracer.configure(priority_sampling=True)
 ```
+
+To set a custom priority to a trace:
+
+```python
+from ddtrace.ext.priority import USER_REJECT, USER_KEEP
+
+context = tracer.context_provider.active()
+
+# indicate to not keep the trace
+context.sampling_priority = USER_REJECT
+```
+
+The following priorities can be used.
+
+|                   Sampling Value                   |                                                   Effect                                                   |
+| -------------------------------------------------- | :--------------------------------------------------------------------------------------------------------- |
+| ddtrace.span.context.sampling_priority.AUTO_REJECT | The sampler automatically decided to not keep the trace. The Agent will drop it.                           |
```

Comment:
why `ddtrace.span.context.sampling_priority`? just list the values because your example above already includes the import path (that is not the one here).